testhelpers/steps_misc.go

```diff
@@ -224,3 +226,40 @@ func (ctx *TestContext) TheLoginModuleAccountEndpointForTokenReturns(token strin
 		httpmock.WithHeader(&http.Header{"Authorization": {"Bearer " + preprocessedToken}})))
 	return nil
 }
+
```

Comment:
These functions are not "misc" anymore, you should move all functions related to loginmodule to a specific file.

Reply:
fixed